### PR TITLE
Fixes #10679: Document fully and partially supported OS

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/10679

### CODE REVIEW

File: 10_installation/05_requirements/23_cloud.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+==== Cloud compatibility
+
+The agent based on CFEngine provides an abstraction that permits a high level management of the infrastructure.
```
Comment (amousset) [inline 10_installation/05_requirements/23_cloud.txt]: We should stay at high level here and not mention underlying CFEngine.

**Comment by peckpeck** [comment]
Commit modified

**Comment by amousset** [comment]
This PR needs to be rebased @peckpeck

**Comment by peckpeck** [comment]
PR rebased

File: 10_installation/05_requirements/30_hardware_specifications.txt [hunk 2]
```diff
@@ -64,4 +48,3 @@ For example, a default installation with 500 nodes and an average of
 
 Follow the <<_reports_retention, Reports Retention>> section to configure the
 retention duration.
```
Comment (amousset) [inline 10_installation/05_requirements/30_hardware_specifications.txt]: Missing empty line

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**